### PR TITLE
added github logo and animations

### PR DESCRIPTION
**Pull request by lopeselio**
made changes to PR #761 .... 
-[x] I have added loop animation to the logo as suggested, used  https://useanimations.com/ to implement it.

file

### CODE REVIEW

**Comment by smaranjitghose** [comment]
## Feedback:

- Please do the following change as well

home_page

**Comment by lopeselio** [comment]
You mean I have to remove the entire subscribe to our newsletter section code along with the enter your email text box ?

**Comment by smaranjitghose** [comment]
> @smaranjitghose You mean I have to remove the entire subscribe to our newsletter section code along with the enter your email text box ?

Yes

**Comment by lopeselio** [comment]
> > @smaranjitghose You mean I have to remove the entire subscribe to our newsletter section code along with the enter your email text box ?
> 
> Yes

image
How does it look ?

**Comment by lopeselio** [comment]
changes have been made

**Review by anushbhatia — CHANGES_REQUESTED** [review]
image
It is not completely centre aligned.

**Comment by lopeselio** [comment]
@anushbhatia @smaranjitghose changes pushed as requested 
image

**Comment by lopeselio** [comment]
Hey @anushbhatia, I have pushed changes to the branch as suggested.

File: src/components/Footer/Footer.js [hunk 1]
```diff
@@ -1,9 +1,12 @@
 import React from "react";
 import { Fade } from "react-reveal";
 import emoji from "react-easy-emoji";
-import { Form } from "react-bootstrap";
-import {FaEnvelope} from "react-icons/fa";
+// import { Form } from "react-bootstrap";
+// import {FaEnvelope} from "react-icons/fa";
```
Comment (smaranjitghose) [inline src/components/Footer/Footer.js:5]: Please do not comment unused code for this project. Simply remove it

**Pull request merged**